sigs.yaml

```diff
@@ -1448,12 +1448,6 @@ sigs:
       - name: Chris Hoge
         github: hogepodge
         company: OpenStack Foundation
-      - name: David Lyle
```

Comment:
Please update the README using `make generate`.

You can also list previous leads as `emeritus_leads` :)

https://github.com/kubernetes/community/blob/08bffde59eab640d3d404029b7f296931a084862/sigs.yaml#L979-L982